tensorflow/lite/micro/kernels/cmsis_nn/fully_connected.cc

```diff
@@ -247,6 +245,70 @@ TfLiteStatus EvalQuantizedInt8(TfLiteContext* context, TfLiteNode* node,
   return kTfLiteOk;
 }
 
+TfLiteStatus EvalQuantizedInt16(TfLiteContext* context, TfLiteNode* node,
+                                const OpData& data,
+                                const TfLiteEvalTensor* input,
+                                const TfLiteEvalTensor* filter,
+                                const TfLiteEvalTensor* bias,
+                                TfLiteEvalTensor* output) {
+  cmsis_nn_per_tensor_quant_params quant_params;
+  quant_params.multiplier = data.reference_op_data.output_multiplier;
+  quant_params.shift = data.reference_op_data.output_shift;
+
+  cmsis_nn_dims input_dims;
+  input_dims.n = data.batches;
```

Comment:
seems line 258 to line 274 are similar with what is in EvalQuantized that they can beneifit from one single function. WDYT?

Reply:
Yes it makes sense

Reply:
Done